TicTacToe2/TicTacToe/Board.cs

```diff
@@ -6,74 +6,41 @@ namespace TicTacToe2.TicTacToe
 {
     class Board
```

Comment:
A few things about the board:

A constructor is the best way to pass identify-defining information to a class - a board can't exist without a size, so the Board class should have a constructor that takes size as an argument.

```c#
public Board(int size)
```

The board is the entity that manages data and functions related to the spaces of a game of tic tac toe. What does the game need to be able to do with the board in a game of tic tac toe?

1. Place a mark on a space
```c#
public void PlaceMark(int x, int y, char symbol)
```

This allows the game to place a mark on the board at the given location. But what happens if the given space is already occupied? You can either give the game the authority to overwrite existing marks with new ones or you can throw an exception (https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/exceptions/creating-and-throwing-exceptions). That said, we would like the game to be smart enough to not try to place marks where it shouldn't. This means that the game will need to have access to data regarding previously placed marks.

2. Check if a mark exists at the given space
```c#
public bool HasMarkAt(int x, int y)
```

The game will also need to be able to check if a space is even on the board

3. Check if a space is on the board
```c#
public bool IsOnBoard(int x, int y)
```

We would also like the board to be able to draw itself

4. Draw board
```c#
public void Draw()
```

Finally, the game needs some way to read the board's spaces to apply game logic - namely, checking for a win or draw. I would like this to be implemented in a specific way that I'll describe later.

So the board's interface (and constructor) look like this:

```c#
public Board(int size)
{
}

public void PlaceMark(int x, int y, char symbol)
{
}

public bool HasMarkAt(int x, int y)
{
}

public bool IsOnBoard(int x, int y)
{
}

public void Draw()
{
}

// TODO: Some function that would give the game access to the data it needs to determine a winner or loser
```